code/__DEFINES/subsystems.dm

```diff
@@ -62,6 +61,7 @@
 #define INIT_ORDER_EVENTS			13
 #define INIT_ORDER_JOBS				12
 #define INIT_ORDER_QUIRKS			11
+#define INIT_ORDER_TICKER			10
```

Comment:
undocumented change

Reply:
I'll document it but it's something you asked me to do long ago